Formula/inxi.rb

```diff
@@ -0,0 +1,20 @@
+class Inxi < Formula
+  desc "Full featured CLI system information tool"
+  homepage "https://smxi.org/docs/inxi.htm"
+  url "https://github.com/smxi/inxi/archive/3.0.38-1.tar.gz"
+  version "3.0.38"
+  sha256 "5adbbe8145e970de1d516a404554e887806d31382de81d290e71fe270183c28a"
+
+  def install
+    bin.install("inxi")
+    man.install("inxi.1")
+
+    ["LICENSE.txt", "README.txt", "inxi.changelog"].each do |file|
+      prefix.install(file)
+    end
+  end
+
+  test do
+    assert `"#{bin}/inxi" --help | grep -- --help` =~ /--help/
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
Sorry, I was again caught by this error.